examples/gno.land/r/system/names/names.gno

```diff
@@ -12,49 +14,71 @@ import (
 // determine if an address can publish a package or not.
 var namespaces avl.Tree // name(string) -> Space
 
+// TODO: more accurate.
+var reNamespace = regexp.MustCompile(`^[a-z][a-z0-9_]{2,30}$`)
```

Comment:
I think we should remove the /r/system from the name space here to prevent adding admin to the /r/system contracts